package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "@nearform/flame",
   "version": "3.5.0",
+  "engines": {
+    "node": "^11.0.0 || ^10.0.0 || >=8.11.1"
```

Comment:
I think this should be `^8.11.1` instead.

Reply:
@mcollina updated in this and other PRs:

- [Bubbleprof](https://github.com/nearform/node-clinic-bubbleprof/pull/300)
- [Clinic](https://github.com/nearform/node-clinic/pull/105)